app.js

```diff
@@ -18,6 +20,7 @@ function testEvent(){
 
   if (answerOne === myAnswerOne) {
     alert('Very well done!');
+    tallyScore = tallyScore + 1;
```

Comment:
Good method of doing this, but you could also just write:

**tallyScore++;** 

That will take the variable and increment it by one. Less code; cleaner looking.